lib/global_id_utils/model_extensions.rb

```diff
@@ -77,14 +77,14 @@ def to_secure_global_id(options = {})
 
     def resolve_gid_app
       return self.class.class_variable_get(:@@gid_app) if self.class.class_variable_defined?(:@@gid_app)
-      return self.class.name.split('::').first.underscore.dasherize if self.class.parent != ::Object
+      return self.class.name.split('::').first.underscore.dasherize if self.class.module_parent != ::Object
```

Comment:
This was causing a deprecation warning. The change to `module_parent` is just the recommended fix in the warning.